doc/tutorial/argument_parser/index.md

```diff
@@ -382,24 +382,46 @@ Add a seqan3::value_list_validator to the `-a/--aggregate-by` option that sets t
 \snippet doc/tutorial/argument_parser/solution6.cpp value_list_validator
 \endsolution
 
-### The seqan3::file_ext_validator
+### The file validator
 
-On construction, the validator receives a list (vector) of valid file extensions.
-The validator throws a seqan3::parser_invalid_argument exception whenever a given filename (string) is not in the given extension list.
+SeqAn3 offers two file validator types: the seqan3::input_file_validator and the seqan3::output_file_validator.
+On construction, the validator receives a list (vector) of valid file extensions that are tested against the extension
+of the parsed option value.
+The validator throws a seqan3::parser_invalid_argument exception whenever a given filename's extension is not in the given list of valid extensions or if the specified path does not name a file but a directory. In addition, the
+seqan3::input_file_validator checks if the file already exists, while on the opposite the seqan3::output_file_validator
+checks if the file does not already exists in order to prevent overwriting an already existing file.
```

Comment:
Note: If you want to allow any extension just pass an empty list to the validator.